src/MICore/LaunchOptions.cs

```diff
@@ -444,6 +444,20 @@ public string VisualizerFile
             }
         }
 
+        private bool _waitDynamicLibLoad;
+        /// <summary>
+        /// If true, wait for dynamic library load to finish.
+        /// </summary>
+        public bool WaitDynamicLibLoad
+        {
+            get { return _waitDynamicLibLoad; }
+            set
+            {
+                VerifyCanModifyProperty("StopOnDynamicLibLoad");
```

Comment:
Not sure I get the question. If the property is false, for linux we might still wait for the first module load to clear the terminal. But other than that we simple not sending the "set stop-on-solib-events 1" command. It should not block debugging.